tests/crypto_test.py

```diff
@@ -148,6 +148,10 @@ def test_luks2_format(self):
         # the simple case with password blob
         succ = BlockDev.crypto_luks_format_blob(self.loop_dev, "aes-xts-plain64", 0, [ord(c) for c in PASSWD], 0)
         self.assertTrue(succ)
+        
+        #test luks2_blob
```

Comment:
You can remove the `BlockDev.crypto_luks_format_blob` call above.

Reply:
ok

Reply:
It has been removed，thanks for your suggestion